include/host/wasi/environ.h

```diff
@@ -1242,6 +1248,13 @@ class EVPoller : protected VPoller {
     }
   }
 
+  void close(__wasi_fd_t Fd) noexcept {
+    if (auto Node = env().getNodeOrNull(Fd); unlikely(!Node)) {
```

Comment:
`if (auto Node = env().getNodeOrNull(Fd); likely(Node)) {`

Reply:
I find I have to use this `if (auto Node = env().getNodeOrNull(Fd); likely((bool)Node))` to compile the code or I will get `error: cannot convert ‘std::shared_ptr<WasmEdge::Host::WASI::VINode>’ to ‘bool’`
Am I doing something wrong?

Reply:
Don't convert it to `bool` directly. `likely(!!Node)` can help.